modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CustomViewsControls.js

```diff
@@ -66,6 +66,34 @@ const CustomViewsControls = () => {
 		);
 	};
 
+	const ActionsItemList = () => {
+		return (
```

Comment:
I recommend you not declare components inside another component closure. Also, since this function just returns something, you can remove the `{` and `}` and remove the `return`

Reply:
See https://github.com/liferay-frontend/liferay-portal/pull/2732#discussion_r974273008

Reply:
Also, good point about no need for `return`!